app/models/container.rb

```diff
@@ -39,6 +42,10 @@ def event_where_clause(assoc = :ems_events)
 
   PERF_ROLLUP_CHILDREN = nil
 
+  def unique_name
```

Comment:
I think a better name would be ```display_name``` that would convey this is ui related
(we use unique names in save_inventory_container to identify a record - ems_ref usually)